src/uct/ib/base/ib_iface.c

```diff
@@ -181,6 +189,13 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "It must not be used together with UCX_IB_GID_INDEX.",
    ucs_offsetof(uct_ib_iface_config_t, rocev2_subnet_filter), UCS_CONFIG_TYPE_ALLOW_LIST},
 
+  {"ROCE_REACHABILITY_MODE", "route",
+   "The mode used for performing the reachability check\n"
+   " - route        - all routable addresses are assumed as reachable\n"
+   " - local_subnet - only addresses within the interface's subnet are assumed as reachable.\n"
+   " - none         - all addresses are assumed as reachable, without any check",
```

Comment:
Maybe `none` -> `all`?
The semantics here suggest the "mode X" means "X address is assumed to be reachable",
thus "mode none" implies "none is reachable", while in fact the opposite is true.